bootloaders/MX1-2-USB-boot-linkerscript-256K.ld

```diff
@@ -0,0 +1,300 @@
+/* Default linker script, for normal executables */
```

Comment:
Is this file a replacement for (i.e., a redundant duplicate of) the MX17-27... linker script for the 256K MX170/MX270?